components/CountryDetails/index.js

```diff
@@ -0,0 +1,67 @@
+import StyledFlag from "../StyledFlag";
+import StyledDiv from "../StyledDiv";
+import Link from "next/link";
+import styled from "styled-components";
+
+const StyledLink = styled.a`
+  font-size: 2rem;
+  position: relative;
+  right: 9rem;
+  top: 1rem;
+`;
+
+export default function CountryDetails({ selectedCountry }) {
+  function addSeparators(number) {
+    return number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ".");
+  }
+  if (!selectedCountry) return <h2>is Loading</h2>;
+  const languagesWithSeparators =
+    selectedCountry.languages &&
+    Object.values(selectedCountry.languages).join(", ");
+
+  const timezoneWithSeparators = selectedCountry.timezones?.join(", ");
+
+  return (
+    <>
```

Comment:
You can remove the fragments here since you are wrapping everything with the `StyledDiv` if I didn't miss anything 🙂